client/src/entrypoints/admin/core.js

```diff
@@ -26,7 +26,7 @@ window.StimulusModule = StimulusModule;
  */
 const wagtail = window.wagtail || {};
 
-/** Initialise Wagtail Stimulus application with core controller definitions. */
+/** Initialize Wagtail Stimulus application with core controller definitions. */
```

Comment:
Maybe this cleanup item can just be a small standalone PR. sorry it was in the previous commits I worked on. Feel free to do this as a separate PR.